### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by julieschwartz18**
Updates based on inputs from @PaulBartruff for PR #1312

## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by julieschwartz18** [comment]
@PaulBartruff - please review and confirm the updates are ok. Thanks!

**Comment by PaulBartruff** [comment]
@julieschwartz18 I can't seem to upload any files here, so I slacked them to you.
I made a number of changes to the instructions to fit the actual language and steps needed to configure the integration, including:
- Changed all the images, which you will still need to put arrows, highlights, or whatever on, but I did label them appropriately for processing.
- Changed from 3 main prerequisites to 4, so as to make the instructions easier to follow.
- Added instructions to add the **Gmail API** in addition to the **Admin SDK API**.

Additionally, when you discover who can review, be sure to note the following:
- Is the Immutable ID a requirement, if so the integration has a bug in the configuration
- According to my testing, both the **Admin SDK API** and the **Gmail API** are required for the integration to work.  Please have the reviewer verify.

**Comment by julieschwartz18** [comment]
@sapirshuker please review the updates to the README file and confirm everything is ok. Specifically:
- Is the Immutable ID mandatory? if so the integration has a bug in the configuration because it let's you leave it empty.
- According to @PaulBartruff 's testing, both the Admin SDK API and the Gmail API are required for the integration to work. Is this correct?
- please confirm the Configure Gmail in Cortex XSOAR and Commands sections are up to date and correct.

**Review by PaulBartruff — APPROVED** [review]
Since I made most the changes, I approve :-), but I really need another technical reviewer to validate the points Julie noted!

**Comment by sapirshuker** [comment]
Hey, @julieschwartz18  @PaulBartruff 
1. The Immutable ID is not mandatory you can configure the instance only with the "Email of user with admin privileges" and "Service account private key file contents (JSON)" you will need "the Immutable Google Apps ID" in specific commands, so it is not mandatory.
2. I do not have permission to check the steps flow or determine the necessity for the Admin SDK API. However, I do know that the Gmail API is required for the 'Service account private key file contents (JSON)'. @julieschwartz18, could you please try @RomanRapoport or Tomer Malache.

**Comment by julieschwartz18** [comment]
> Hey, @julieschwartz18 @PaulBartruff
> 
> 1. The Immutable ID is not mandatory you can configure the instance only with the "Email of user with admin privileges" and "Service account private key file contents (JSON)" you will need "the Immutable Google Apps ID" in specific commands, so it is not mandatory.
> 2. I do not have permission to check the steps flow or determine the necessity for the Admin SDK API. However, I do know that the Gmail API is required for the 'Service account private key file contents (JSON)'. @julieschwartz18, could you please try @RomanRapoport or Tomer Malache.

Thanks @sapirshuker , I'll add them as reviewers as well.

**Comment by julieschwartz18** [comment]
I opened a Jira ticket [CRTX-80465](https://jira-hq.paloaltonetworks.local/browse/CRTX-80465) to review the updated Gmail integration README. According to Tomer, his Team Leader Shai Padan will review and assign.

**Comment by sapirshuker** [comment]
Hey, @PaulBartruff In addition, I think we don't need the 'Admin SDK API' for all the integration commands, need to add to the documentation for which commands it needed.

**Comment by PaulBartruff** [comment]
@sapirshuker I believe the test fails without the admin-sdk.  However, my testing was limited.

R/
Paul

**Comment by julieschwartz18** [comment]
Based on @RomanRapoport 's review of the documentation, merging the updates with the master.

**Review by RomanRapoport — APPROVED** [review]
All the mentioned scopes in the doc are configured in the service account's domain-wide delegation.

File: Packs/Gmail/Integrations/Gmail/README.md [hunk 1]
```diff
@@ -4,52 +4,54 @@ This integration replaces the Gmail functionality in the GoogleApps API and G Su
...
+7.  In the **Grant users access to this service account** section, click **DONE**.</br>
+8.  In the **Actions** column for the newly created service account, click the verticle elipses, then click **Manage keys**.</br>![gmail_section1_step8](https://raw.githubusercontent.com/demisto/content-docs/c8ff74615d254fcb4f4f0caf3d2a00da156b8b04/docs/doc_imgs/integrations/gmail_section1_step8.png)   
+9. Click the **ADD KEY** dropdown, and select **Create new key**.
+10. Select Key type **JSON** and click **CREATE**.</br>This will generate a json **Private key** file that will be downloaded and saved locally.
+11. Click **CLOSE** to close the dialog.
+12. Navigate to **DETAILS** -> **Advanced Settings**.  Copy the **Client ID**.
+
+### Enable APIs
+
+Both the **GMail API** and the **Admin SDK API** are required to use this integration.
```
Comment (sapirshuker) [inline Packs/Gmail/Integrations/Gmail/README.md:33]: **Gmail API**

Comment (julieschwartz18) [reply Packs/Gmail/Integrations/Gmail/README.md]: Fixed, thanks!

**Review by sapirshuker — APPROVED** [review]

**Pull request merged**